Overall review of Core docs 3.0.0 - Closes #677 (#680)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/commander.adoc

== Post-install

> After the installation is completed, check which ports Lisk Core is listening to by checking the status using the applicable command shown below:

Reviewer: ```suggestion
After the installation is completed, check which ports Lisk Core is listening to.
To do that, check the status of Lisk Core by using the command shown below:
```

--
====

> Check your network settings to verify the corresponding ports are open.

Reviewer: ```suggestion
Then check the network settings to verify the corresponding ports are open.
```

In addition it is also recommended to set up a xref:{url_config_logrotation}[log rotation].

> If you are not running Lisk locally, you will need to follow the xref:{url_config_api_access}[API - Configuration] guide to enable access.

Reviewer: ```suggestion
If you are not running Lisk Core locally, you will need to follow the xref:{url_config_api_access}[Control API access] guide to enable access.
```
